services/order_by.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.utils.translation.trans_real import translation
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list[dict], username: str, date: datetime) -> list[dict]:
+    with translation.atomic():
+        order = Order.objects.create(username=username, created_at=date)
+        for ticket in tickets:
+            Ticket.objects.create(row=ticket["row"],
+                              seat=ticket["seat"],
+                              movie_session=ticket["movie_session"],
+                              order=order)
+
+
+def get_orders(username: str = None) -> QuerySet[Order]:
```

Comment:
The return type hint `QuerySet[Order]` is incorrect. Django's `QuerySet` does not support generics in this way. It should be just `QuerySet`.